setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 [flake8]
 application-import-names = orderable
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D105,D203,D204
```

Comment:
Though actually, there's still a lot of errors we do catch: http://pep257.readthedocs.org/en/latest/error_codes.html